readme.md

```diff
@@ -110,17 +110,17 @@ await $({stdio: 'inherit'})`echo unicorns`;
 //=> 'unicorns'
 ```
 
-#### Shared options
+#### Global/shared options
 
 ```js
-import {$} from 'execa';
+import {$ as $_} from 'execa';
 
-const $$ = $({stdio: 'inherit'});
+const $ = $_({stdio: 'inherit'});
 
-await $$`echo unicorns`;
+await $`echo unicorns`;
 //=> 'unicorns'
 
-await $$`echo rainbows`;
+await $`echo rainbows`;
 //=> 'rainbows'
 ```
```

Comment:
This file also includes [an example](https://github.com/sindresorhus/execa/?tab=readme-ov-file#options) of `const $$ = $(options)` for shared options. It shows the following text, which did not need to be changed:

```
This can be used to either:
  - Set options for a specific command: `$(options)`command` `
  - Share options for multiple commands: `const $$ = $(options); $$`command`; $$`otherCommand`;`
```